ui/kotlinx-coroutines-javafx/src/JavaFxConvert.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright 2016-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.coroutines.javafx
+
+import javafx.beans.value.ChangeListener
+import javafx.beans.value.ObservableValue
+import kotlinx.coroutines.*
+import kotlinx.coroutines.channels.awaitClose
+import kotlinx.coroutines.flow.*
+
+/**
+ * Creates an instance of a cold [Flow] that subscribes to the given [ObservableValue] and produces
```

Comment:
KDoc wording: Flow does not *produce*, it *emits*

Reply:
`a quick succession` -> `quick succession`

Reply:
A lot of newlines are not really necessary

Reply:
I took the word "produce" from https://kotlinlang.org/docs/reference/coroutines/flow.html where it is used not only in the context of producing an output of the whole program, so maybe it, too, should be fixed there.

In any case, I committed an improved KDoc.